vizro-core/examples/bootstrap/app.py

```diff
@@ -0,0 +1,890 @@
+                        body=True,
+                    ),
+                    label="Tab 2",
+                    style={"padding": "10px"},
+                ),
+            ]
+        ),
+    ],
+    className="mb-4",
+)
+
+toast = html.Div(
+    [
+        make_subheading("dbc.Toast", "toast"),
+        dbc.Button(
+            "Open toast",
+            id="dbc-gallery-x-auto-toast-toggle",
+            color="primary",
+            className="mb-3",
+        ),
+        dbc.Toast(
+            html.P("This is the content of the toast", className="mb-0"),
+            id="dbc-gallery-x-auto-toast",
+            header="This is the header",
+            icon="primary",
+            duration=4000,
+        ),
+    ],
+    className="mb-2",
+)
+
+
+@callback(
+    Output("dbc-gallery-x-auto-toast", "is_open"),
+    [Input("dbc-gallery-x-auto-toast-toggle", "n_clicks")],
+)
+def open_toast(_):
+    """Make toast reactive."""
+    return True
+
+
+tooltip = html.Div(
+    [
+        make_subheading("dbc.Tooltip", "tooltip"),
+        html.P(
+            [
+                "I wonder what ",
+                html.Span("floccinaucinihilipilification", id="gallery-tooltip-target"),
+                " means?",
+            ]
+        ),
+        dbc.Tooltip(
+            "Noun: rare, " "the action or habit of estimating something as worthless.",
+            target="gallery-tooltip-target",
+        ),
+    ],
+    className="mb-4",
+)
+
+
+layout = html.Div(
+    [
+        alerts,
+        badges,
+        buttons,
+        cards,
+        collapse,
+        fade,
+        dbc.Row(
+            [
+                dbc.Col([form, input_group], xs=12, md=6),
+                dbc.Col([input_], xs=12, md=6),
+            ]
+        ),
+        dbc.Row(
+            [
+                dbc.Col([checklist_items], xs=12, md=6),
+                dbc.Col([radio_items], xs=12, md=6),
+            ]
+        ),
+        list_group,
+        modal,
+        navbar,
+        popover,
+        progress,
+        spinner,
+        table,
+        tabs,
+        toast,
+        tooltip,
+    ],
+    className="dbc",
+)
+
+color_mode_switch = html.Span(
+    [
+        dbc.Label(className="fa fa-moon", html_for="color-mode-switch"),
+        dbc.Switch(id="color-mode-switch", value=False, className="d-inline-block ms-1", persistence=True),
+        dbc.Label(className="fa fa-sun", html_for="color-mode-switch"),
+    ]
+)
+
+app.layout = dbc.Container(
+    [
+        html.H3(["Bootstrap Light Dark Color Modes Demo"], className="bg-primary p-2"),
+        color_mode_switch,
+        layout,
+    ]
+)
+
+
+clientside_callback(
+    """
+    (switchOn) => {
+       document.documentElement.setAttribute('data-bs-theme', switchOn ? 'light' : 'dark');
+       return window.dash_clientside.no_update
+    }
+    """,
+    Output("color-mode-switch", "id"),
+    Input("color-mode-switch", "value"),
+)
+
+
+if __name__ == "__main__":
+    app.run_server(debug=True)
```

Comment:
```suggestion
    app.run()
```

When Dash 3 is released, `run_server` is removed.

Reply:
Just to check: does `hatch run example bootstrap` work? So long as the example file lives here, ideally it would.

Reply:
Yes, that works 👍